src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/DocumentControllerTest.kt

```diff
@@ -355,4 +369,31 @@ class DocumentControllerTest {
     verify { revocationOrderService wasNot Called }
     verify { reasonsForRecallService wasNot Called }
   }
+
+  @Test
+  fun `generate new document without details`() {
```

Comment:
I think I'd prefer less duplication between these tests and instead re-use of a shared method - but for another time.